sdk/core/azure-core-tracing-opentelemetry/README.md

```diff
@@ -50,33 +50,34 @@ The following sections provides examples of using the azure-core-tracing-opentel
     This [sample][sample_key_vault] provides an example when no user parent span is passed.
 
     ```java
-    private static  final Tracer TRACER;
-    private static final TracerSdkFactory TRACER_SDK_FACTORY;
+    private static  final Tracer TRACER = TRACER_SDK_FACTORY.get("Sample");
```

Comment:
Can you also update this to use readme snippets tool so that we will have compilable code?